openid-4-verifiable-credential-issuance-1_0.md

```diff
@@ -694,13 +694,16 @@ Cache-Control: no-store
 
 # Credential Endpoint {#credential-endpoint}
 
-The Credential Endpoint issues a Credential as approved by the End-User upon presentation of a valid Access Token representing this approval. Support for this endpoint is REQUIRED.
+The Credential Endpoint issues one or more instances of a Credential Configuration or Credential Dataset as approved by the End-User upon presentation of a valid Access Token representing this approval. Support for this endpoint is REQUIRED.
```

Comment:
```suggestion
The Credential Endpoint issues one or more instances of a Credential Dataset as approved by the End-User upon presentation of a valid Access Token representing this approval. Support for this endpoint is REQUIRED.
```

Credential configuration is merely how credential is described, what is actually issued is credential dataset.

Reply:
I'm not certain about the proposed changes, I'd like to get more feedback on this from others

Reply:
Batch Endpoint says `each of these objects may refer to a different Credential Configuration or the same Credential Configuration with a different Credential Dataset`

```suggestion
The Credential Endpoint issues one or more Credentials of the same Credential Configuration and Credential Dataset, as approved by the End-User upon presentation of a valid Access Token representing this approval. Support for this endpoint is REQUIRED.
```